collector/collector.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"runtime"
+	"time"
+
+	"github.com/AstroProfundis/sysinfo"
+)
+
+type meta struct {
```

Comment:
An unexported struct with exported members. Do you have any special considerations here？

Reply:
Rest LGTM

Reply:
Nothing special here, I'll correct this later.